systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectST.java

```diff
@@ -1724,6 +1730,175 @@ void testKafkaConnectWithScramShaAuthenticationRolledAfterPasswordChanged() {
+                .addToConfig("value.converter", "org.apache.kafka.connect.storage.StringConverter")
+                .addToConfig("topics", testStorage.getTopicName())
+                .withNewAlterOffsets()
+                    .withNewFromConfigMap(offsetConfigMap)
+                .endAlterOffsets()
+                .withNewListOffsets()
+                    .withNewToConfigMap(offsetConfigMap)
+                .endListOffsets()
+            .endSpec()
+            .build();
+
+        resourceManager.createResourceWithWait(
+            connect,
+            kafkaConnector,
+            ScraperTemplates.scraperPod(testStorage.getNamespaceName(), testStorage.getScraperName()).build()
+        );
+
+        LOGGER.info("Deploying network policies for KafkaConnect");
+        NetworkPolicyResource.deployNetworkPolicyForResource(connect, KafkaConnectResources.componentName(testStorage.getClusterName()));
+        final String scraperPodName = PodUtils.getPodsByPrefixInNameWithDynamicWait(testStorage.getNamespaceName(), testStorage.getScraperName()).get(0).getMetadata().getName();
+
+        KafkaClients kafkaClients = ClientUtils.getInstantPlainClients(testStorage);
+
+        resourceManager.createResourceWithWait(kafkaClients.producerStrimzi(), kafkaClients.consumerStrimzi());
+
+        KafkaConnectorUtils.waitForOffsetInFileSinkConnector(testStorage.getNamespaceName(), scraperPodName, testStorage.getClusterName(), testStorage.getClusterName(), TestConstants.MESSAGE_COUNT);
+
+        LOGGER.info("Listing the Connector's offsets using the {} annotation set to 'list'", ResourceAnnotations.ANNO_STRIMZI_IO_CONNECTOR_OFFSETS);
+
+        // annotate the KafkaConnector resource to list the offsets of the file sink connector
+        KafkaConnectorResource.replaceKafkaConnectorResourceInSpecificNamespace(testStorage.getNamespaceName(), testStorage.getClusterName(),
+            connector -> connector.getMetadata().getAnnotations().put(ResourceAnnotations.ANNO_STRIMZI_IO_CONNECTOR_OFFSETS, "list"));
+
+        // wait for creation of the CM
+        ConfigMapUtils.waitForCreationOfConfigMap(testStorage.getNamespaceName(), offsetConfigMap);
+
+        // checking the config map
+        ConfigMap offsetsConfigMap = kubeClient().getConfigMap(testStorage.getNamespaceName(), offsetConfigMap);
+        JsonNode offsets = mapper.readTree(offsetsConfigMap.getData().get("offsets.json"));
+
+        assertThat("Offset config map contains correct kafka_offset value", offsets.at("/offsets/0/offset/kafka_offset").asInt(), is(TestConstants.MESSAGE_COUNT));
+
+        LOGGER.info("Changing the offset in the ConfigMap to {}, stopping the Connector and applying the {} annotation with 'alter'", desiredNewOffset, ResourceAnnotations.ANNO_STRIMZI_IO_CONNECTOR_OFFSETS);
+
+        // set the offsets to `20`
+        ((ObjectNode) offsets.get("offsets").get(0).get("offset")).put("kafka_offset", desiredNewOffset);
+
+        // update the connector to be stopped
```

Comment:
can it be mentioned that the connector actually need to be stopped in order to alter offset work properly, either here in doc  ?

Reply:
it's in the steps and the documentation of the test.
https://github.com/strimzi/strimzi-kafka-operator/pull/10904/files#diff-5223a7b668a476b0e7b142b5bfd42745ba845df92d9d8722397d3510e514ceb3R1739

Or do you want to have something more explicit?